connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -40,6 +41,260 @@ import logger from "@connectors/logger/logger";
 import type { DataSourceConfig } from "@connectors/types/data_source_config";
 import type { GoogleDriveObjectType } from "@connectors/types/google_drive";
 
+async function handleGoogleDocExport(
```

Comment:
This looks great 🔥 Can you rebase on the latest changes with Tika 🙏

Reply:
This function should be called `handleGoogle[Drive]Export()`. The Google doc API is a different one.

Reply:
Or for consistency with `handleFileDownload()`, we can also call it `handleFileExport()`

Reply:
✅ Used Tika for pptx, .docx and pdf